### PR TITLE
Air fryer

### PR DESCRIPTION
**Pull request by Sthopeless**
Adding support for Air Fryer (kqzg) 

Switches (Turn On/Off and Start/Pause and Preheat On/Off)

https://developer.tuya.com/en/docs/iot/f?id=Kakdaoinr5xlu

### CODE REVIEW

**Comment by Sthopeless** [comment]
Zigbee Doorlock (ms) Doorbell Volume 

Screenshot (128)

File: custom_components/tuya_v2/select.py [hunk 1]
```diff
@@ -1,19 +1,18 @@
 #!/usr/bin/env python3
```
Comment (Jhonattan-Souza) [inline custom_components/tuya_v2/select.py:1]: I think you've made a mistake here, this is not the number entity file, is the SelectEntity file, you're removing functionalities from the library

**Review by Jhonattan-Souza — CHANGES_REQUESTED** [review]

File: custom_components/tuya_v2/select.py (hunk 1 above)
Comment (Sthopeless) [reply custom_components/tuya_v2/select.py]: sorry somehow confused one file with the other.. fixed below

File: custom_components/tuya_v2/switch.py [hunk 2]
```diff
@@ -41,19 +42,23 @@
...
+DPCODE_WET = "wet"             # Air Purifier - Humidification unit
 DPCODE_PRESET = "pump_reset"   # Pet Water Feeder - Water pump resetting
 DPCODE_WRESET = "water_reset"  # Pet Water Feeder - Resetting of water usage days
 
 
 DPCODE_START = "start"
+# Air Fryer
+# https://developer.tuya.com/en/docs/iot/f?id=Kakdaoinr5xlu
+DPCODE_AIRFRYER_PREHEAT = "preheat"
+DPCODE_AIRFRYER_SPIN = "spin_switch"
```
Comment (tsutsuku) [inline custom_components/tuya_v2/switch.py:61]: According to doc spin_switch data type is Integer, why put it in the switch.py?

Comment (Sthopeless) [reply custom_components/tuya_v2/switch.py:61]: @tsutsuku I noticed documentation said integer but at least on mine testing Air Fryer is shows as a boolean, check last option:
Screenshot_2021-07-23-07-24-09-009_com tuya smartlife

**Comment by Sthopeless** [comment]
this is enabled no other option/configuration available for me
Screenshot_2021-07-23-08-16-15-976_com tuya smartlife

**Comment by Sthopeless** [comment]
@tsutsuku is the `spin_switch` being checked?  
In Home Assistant if I set it to integer nothing happens it only works as a boolean switch.

Smartlife only has On/Off option.

On [cloud explorer](https://iot.tuya.com/cloud/explorer) I see it as a integer 
but I try to change the value I get a error saying `type is incorrect` also nothing happens in Smartlife app or Home Assistant logs

**Pull request closed without merging**